docs/content/releases.md

```diff
@@ -6,6 +6,69 @@ doctypes: ["concept"]
 toc: true
 docs: "DOCS-616"
 ---
+## NGINX Ingress Controller 2.3.0
+
+12 July 2022
+
+OVERVIEW:
+
+* Support making VirtualServer resources discoverable via public DNS servers using [ExternalDNS](https://kubernetes-sigs.github.io/external-dns). Examples for configuring ExternalDNS with NGINX Ingress Controller can be found [here](https://github.com/nginxinc/kubernetes-ingress/tree/v2.3.0/examples/custom-resources/external-dns).
```

Comment:
this should be corrected to external-dns instead of ExternalDNS - probably my bad since this is auto-generated.